bindings/rust-primitives/src/rollup_config.rs

```diff
@@ -32,7 +57,7 @@ pub const OP_SEPOLIA_EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u128 = 6;
 
 /// Base fee max change denominator for Base Sepolia as defined in the Optimism
 /// [transaction costs](https://community.optimism.io/docs/developers/build/differences/#transaction-costs) doc.
-pub const BASE_SEPOLIA_EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u128 = 10;
+pub const BASE_SEPOLIA_EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u128 = 6;
```

Comment:
This is not right. Base sepolia has a different elasticity multiplier at 10; I fixed this a few weeks back. See https://github.com/paradigmxyz/reth/blob/8c8702be14c6811a4166c279ca1f8df92308044b/crates/primitives-traits/src/constants/mod.rs#L89-L91 + https://github.com/ethereum-optimism/superchain-registry/pull/311

Reply:
Strange, the scr reference config says 6 - will dig in